src/featureFlags/flags/swapWidget.ts

```diff
@@ -1,11 +1,14 @@
+import { useGate } from 'statsig-react'
+
 import { BaseVariant, FeatureFlag, useBaseFlag } from '../index'
 
 export function useSwapWidgetFlag(): BaseVariant {
   return useBaseFlag(FeatureFlag.swapWidget, BaseVariant.Control)
 }
 
 export function useSwapWidgetEnabled(): boolean {
-  return useSwapWidgetFlag() === BaseVariant.Enabled
+  const { value: statsigValue } = useGate(FeatureFlag.swapWidget)
```

Comment:
Can this be moved into `useBaseFlag`, which `useBaseFlag` taking an optional parameter with the gate name?